### PR TITLE
Create CODE-OF-CONDUCT.md

### PR DESCRIPTION
**Pull request by Trueblueaddie**
This is a requested document for the repos going to the MBO. It follows a standard format.

### CODE REVIEW

**Review by JaredCorduan — APPROVED** [review]
Thanks @Trueblueaddie ! You've done half of  #3119 for us. :)

**Comment by JaredCorduan** [comment]
@Trueblueaddie - I think I'll hold off merging this, until we resolve the discussions in https://github.com/input-output-hk/ouroboros-network/pull/4133 (and I'm leaning more and more towards just linking to a CoC hosted by IOG).

**Comment by Trueblueaddie** [comment]
That makes a lot of sense @JaredCorduan

File: CODE-OF-CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+Project maintainers who do not follow or enforce the Code of Conduct in good
+faith may face temporary or permanent repercussions as determined by other
+members of the project's leadership.
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
+available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html
+
+[homepage]: https://www.contributor-covenant.org
```
Comment (JaredCorduan) [inline CODE-OF-CONDUCT.md:73]:
```suggestion
See the code of conduct in the [https://github.com/input-output-hk/cardano-engineering-handbook/blob/main/CODE-OF-CONDUCT.md](Cardano engineering handbook).
```

cc @Trueblueaddie

**Comment by JaredCorduan** [comment]
resolved by #3139

**Pull request closed without merging**